OSDOCS#14993: Adding monitoring cert-manager metrics section (#95058)
openshift/openshift-docs: modules/cert-manager-query-metrics.adoc

:_mod-docs-content-type: PROCEDURE
[id="cert-manager-query-metrics_{context}"]
= Querying metrics for the cert-manager operands

As a cluster administrator, or as a user with view access to all namespaces, you can query cert-manager operands metrics by using the {product-title} web console or the command-line interface (CLI). For more information, see "Accessing metrics".

.Prerequisites

* You have access to the cluster as a user with the `cluster-admin` role.
* You have installed the {cert-manager-operator}.
* You have enabled monitoring and metrics collection by creating `ServiceMonitor` object.

.Procedure

. In the {product-title} web console, navigate to *Observe* → *Metrics*.

> . In the *Label* filter field, enter the following commands to query the cert-manager operands metric targets for each operand:

Reviewer: ```suggestion
. In the query field, enter the following PromQL expressions to query the cert-manager operands metric for each operand:
```

+

> [source,terminal]

Reviewer: ```suggestion
[source,promql]
```

----
$ service=cert-manager
----
+
[source,terminal]
----
$ service=cert-manager-webhook
----
+
[source,terminal]
----
$ service=cert-manager-cainjector
----

> . Confirm that the *Status* column shows `Up` for the `cert-manager`, `cert-manager-webhook`, and `cert-manager-cainjector` entries.

Reviewer: ```suggestion
```
